haxepunk/graphics/hardware/DrawCommand.hx

```diff
@@ -154,26 +153,45 @@ class DrawCommand
 
 	function new() {}
 
-	public inline function match(texture:BitmapData, shader:Shader, smooth:Bool, blend:BlendMode, clipRect:Rectangle):Bool
+	/**
+	 * Compares values to this draw command to see if they all match. This is used by the batcher to reuse the previous draw command.
+	 */
+	public inline function match(texture:Texture, shader:Shader, smooth:Bool, blend:BlendMode, clipRect:Rectangle):Bool
```

Comment:
I was surprised by this as well but it seems to inline on my mac. Although I didn't check the javascript output.